galette/templates/default/page.html.twig

```diff
@@ -23,18 +24,12 @@
                             <div class="sub header">{% if preferences.pref_slogan %}{{ __(preferences.pref_slogan) }}{% endif %}</div>
                         </div>
                     </div>
+                    <a id="main-content" tabindex="-1"></a>
                     <h1 class="ui block center aligned header" style="position: relative">
 {% else %}
                     <div class="ui horizontal basic segments header">
-                        <div class="ui center aligned segment">
-                            <h1>
-{% endif %}
-                                {% block page_title %}{{ page_title }}{% endblock %}
-                            </h1>
-{% if login.isLogged() %}
-                        </div>
                         <div class="ui right aligned compact segment">
-                            <div class="ui compact tiny menu">
+                            <nav class="ui compact tiny menu" aria-label="{{ _T('Secondary menu') }}">
```

Comment:
This is not a menu on a semantic point of view
```suggestion
                            <nav class="ui compact tiny menu">
```

Reply:
It would if it only contained the language menu and was labeled as such.
But because it can show an existing email "notification" link, I chose to name it that way.

What if I move the existing email link elsewhere ? :thinking:

Reply:
OK, I guess I understand... 

Existing emailing can be moved, that's not a main feature (and we also get reminder a mailing already exists when creating a new one from members list).
I guess we can move this as an extra entry in main menu just before/after "Manage mailings".

Reply:
(or fource, it also can be moved elsewhere if you have a better idea :D)

Reply:
> (of course, it also can be moved elsewhere if you have a better idea :D)

You'll judge if it's better, but yes I have an other idea :wink:

Reply:
> > (of course, it also can be moved elsewhere if you have a better idea :D)
> 
> You'll judge if it's better, but yes I have an other idea 😉

I moved it on the top left corner of the the screen, as the first link in the navigation (or in the banner for the mobile menu) with a pulsating transition.

On mobile : 
![Capture d’écran de 2023-11-20 21-42-23](https://github.com/galette/galette/assets/107203963/8b6d19fb-7914-4a31-9c87-5ede849afa39)

Logged in with the default menu :
![Capture d’écran de 2023-11-20 21-40-49](https://github.com/galette/galette/assets/107203963/db5b846f-8eed-43ec-98b1-49318743af37)

Logged in with the compact menu :
![Capture d’écran de 2023-11-20 21-41-25](https://github.com/galette/galette/assets/107203963/5ba4e735-4c4f-4668-ba39-54269492ee10)

Reply:
Well... That's not a so bad idea; but I'm not fan, sorry :/

It make looks like there is something important happening, but in fact that only means there is an mailing in session that has not been stored... And that is not really a feature, but a kind of technical fact.

The existing mailing button is really not something important. Maybe it was at the very beginning, but now users cans store their current mailing in database and retrieve it later; disregarding the session.

For me, this button can also just be removed.

Reply:
> For me, this button can also just be removed.

All right! I agree :+1: This non-feature is confusing :sweat_smile: